api/server.go

```diff
@@ -406,14 +405,13 @@ type loggingResponseWriter struct {
 }
 
 func (w *loggingResponseWriter) Write(data []byte) (int, error) {
-	fmt.Println("<-----", string(data))
 	body := make(map[string]any)
 	_ = json.Unmarshal(data, &body)
 
 	w.logger.
 		Debug().
-		Str("id", fmt.Sprintf("%v", body["id"])).
-		Str("result", fmt.Sprintf("%v", body["result"])).
+		Float64("id", body["id"].(float64)).
+		Fields(body["result"]).
```

Comment:
The `body["result"]` is not always a container value (e.g. an array or map). It can also be a scalar value, e.g.

```json
{
  "jsonrpc": "2.0",
  "id": 1,
  "result": "0x2"
}
```

So the `Fields` will not work for this case.